skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2007 Zachary Pincus
+# This file is part of CellTool.
+#
+# CellTool is free software; you can redistribute it and/or modify
+# it under the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
+
+from scipy import ndimage
+import numpy
+
+def warp_images(from_points, to_points, images, output_region, interpolation_order = 1, approximate_grid=2):
```

Comment:
Do we want the function to take in a *list* of images as opposed to a single image? I think a single image makes more sense, but there could be a use-case where taking a list of images as input is essential.

Reply:
I think the API we will come up with should support both use cases easily. It might be useful to expose what is currently `_make_inverse_warp` as a low-level public function so that users can customize the algorithm if the standard `warp_images` is not enough. E.g. the output of `_make_inverse_warp` can already be passed to `skimage.transform.warp` after transforming it into an array.

Reply:
That's interesting, we could look into that.